packages/cozy-harvest-lib/test/components/KonnectorConfiguration/DataTab.spec.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import { DataTab } from 'components/KonnectorConfiguration/DataTab'
+import LaunchTriggerCard from 'components/cards/LaunchTriggerCard'
+import DocumentsLinkCard from 'components/cards/DocumentsLinkCard'
+
+describe('DataTab', () => {
+  beforeEach(() => {
+    jest.spyOn(console, 'error').mockImplementation(() => {})
```

Comment:
Failing PropTypes because of the testing context, I'll add a comment.

Reply:
Can't we fix the propType issue instead of hiding it?

Reply:
The issue is that a child component (not even the component we're testing here) needs a translation context, and that would require using `mount` instead of `shallow`, so needing to mock a lot more things.

Reply:
In Banks, it is common to wrap our components into a I18n component, it also helps since snapshots have the right translations inside them.

Reply:
But then you have to use `mount` and not `shallow`, right?

(no snapshots here 😃)

Reply:
Personally I prefer the mount versus the hide of errors

Reply:
> But then you have to use mount and not shallow, right?

Why ?

I agree with @Crash-- here.

Reply:
If I do `shallow(<I18n><MyComponent /></I18n>)` I need to `dive`, and thenthe proptype errors appear again. Same goes if I provide a `wrappingComponent` to `shallow`.

> Personally I prefer the mount versus the hide of errors

Mounting on such a high level component is a bit of a nightmare, really. If the errors came from the component we're testing I would be on board with this, but they aren't.

Reply:
Is it possible to do

```
jest.mock('ChildComponent', () => () => null) non ?
```


The issue is that a child component (not even the component we're testing here) needs a translation context, and that would require using mount instead of shallow, so needing to mock a lot more things.